examples/tv-app/android/java/MediaPlaybackManager.cpp

```diff
@@ -220,6 +220,38 @@ uint64_t MediaPlaybackManager::HandleMediaRequestGetAttribute(MediaPlaybackReque
     return ret;
 }
 
+long MediaPlaybackManager::HandleMediaRequestGetLongAttribute(MediaPlaybackRequestAttribute attribute)
+{
+    long ret              = 0;
+    jlong jAttributeValue = -1;
+    CHIP_ERROR err        = CHIP_NO_ERROR;
+    JNIEnv * env          = JniReferences::GetInstance().GetEnvForCurrentThread();
+
+    ChipLogProgress(Zcl, "Received MediaPlaybackManager::HandleMediaRequestGetLongAttribute:%d", attribute);
+    VerifyOrExit(mMediaPlaybackManagerObject != nullptr, err = CHIP_ERROR_INCORRECT_STATE);
+    VerifyOrExit(mGetAttributeMethod != nullptr, err = CHIP_ERROR_INCORRECT_STATE);
+    VerifyOrExit(env != NULL, err = CHIP_JNI_ERROR_NO_ENV);
+
+    jAttributeValue = env->CallLongMethod(mMediaPlaybackManagerObject, mGetAttributeMethod, static_cast<jint>(attribute));
+    if (env->ExceptionCheck())
+    {
+        ChipLogError(AppServer, "Java exception in MediaPlaybackManager::GetAttribute");
```

Comment:
should this not set some error in ERR if we get an exception?

It seems without that, in case of exception this returns 0 ... is that intended?

Some doccomment describing what this does and expectation may help